core/common/base.py

```diff
@@ -23,17 +23,18 @@ class BaseSocketServer:
     """
 
     def __init__(self, soc: socket.socket):
-        """
+        """Initialize BaseSocketServer Object to soc.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)